tests/parser/functions/test_convert_int128.py

```diff
@@ -1,8 +1,61 @@
 from vyper.exceptions import (
+<<<<<<< HEAD
```

Comment:
Checked in a merge conflict!

Reply:
@fubuloubu I believe that's on an old commit 😅I messed up and committed `8bfebe1` before I should have but fixed with `caaae76` which is latest. I can squash though all the commits here if you'd like and update the history?

Reply:
Yep, squash it plz!

Reply:
Haha already did! 😅